src/Package/Impl/DataInspect/EvaluationWrapper.cs

```diff
@@ -148,5 +153,16 @@ public sealed class EvaluationWrapper : RSessionDataObject, IIndexedItem {
             return false;
         }
         #endregion
+
+        private bool ComputCsvAvailability(DebugValueEvaluationResult evaluation) {
```

Comment:
Open in CSV also handles vectors. Grid wants 2-dimensional and I am not sure it is safe to enable vectors in the grid (although we have user request for that).